src/Application.php

```diff
@@ -54,6 +54,8 @@ private function createContainer(): ContainerBuilder
         $loader->load('services.xml');
         $container->compile();
 
+        (new Filesystem())->dumpFile(\dirname(__DIR__) . '/var/cache/container.xml', (new XmlDumper($container))->dump());
```

Comment:
you dump into vendor folder, maybe dump only with a environment variable

Reply:
good hint! you are right, I will change that

Reply:
done